docker/charts/templates/controller-deployment.yaml

```diff
@@ -67,7 +75,7 @@ spec:
               value: controller
               {{- if not .Values.minikube.enabled }}
             - name: JVM_XMX
-              value: {{ .Values.resources.controller.requests.memory }}
+              value: {{ .Values.resources.controller.limits.memory }}
```

Comment:
Maybe we could remove this block?
ref: https://www.eclipse.org/openj9/docs/xxusecontainersupport/

Reply:
yes,done